jsonschema-module-jakarta-validation/src/test/java/com/github/victools/jsonschema/module/jakarta/validation/IntegrationTest.java

```diff
@@ -128,5 +131,27 @@ static class TestClass {
         @DecimalMin(value = "0", inclusive = false)
         @DecimalMax(value = "1", inclusive = false)
         public double exclusiveRangeDouble;
+
+        @PositiveOrZero
+        public IntegrationTest.TestSupplier supplierPositiveOrZero;
+
+        public TestSupplierWithAnnotation supplierWithAnnotationPositiveOrZero;
+    }
```

Comment:
**suggestion**:  The tests should also consider a field with type `Supplier<@PositiveOrZero Integer>`.
Also a scenario in which the actual getter method holds the annotation, e.g.
```java
@PositiveOrZero
public Supplier<Integer> getSupplierPositiveOrZeroWithGetter() {
    return this.supplierPositiveOrzeroWithGetter;
}
```